webpack.config.js

```diff
@@ -8,7 +8,7 @@ const base = {
         host: '0.0.0.0',
         port: process.env.PORT || 8361
     },
-    devtool: 'source-map',
+    devtool: 'cheap-module-source-map',
```

Comment:
This option has treated me well over the past few days, though I'm happy to discuss and experiment with other options.
See also: https://webpack.js.org/configuration/devtool/#devtool